lib/order.rb

```diff
@@ -0,0 +1,59 @@
+# I would like to be able to select some number of several available dishes
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+require_relative 'menu.rb'
+
+class Order
+
+attr_reader :basket, :balance
+
+  def initialize(menu= Menu.new)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `menu`. If it's necessary, use `_` or `_menu` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)